lib/tasks/deploy.rake

```diff
@@ -2,6 +2,7 @@ GITHUB_ORG = 'ad2games'
 BASE_IMAGE = 'docker-rails:latest'
 DEPLOY_USER = 'ad2gamesdeploy'
 DEPLOY_EMAIL = 'developers@ad2games.com'
+DEPLOY_ENVS = { 'staging' => %w(staging) } # N.b., master is not deployed by default
```

Comment:
Right, that's true, too.  I just made it an empty hash.  That way auto-deployment feature is still optional, and setting `DEPLOY_ENVS` is just required for it to trigger any deployment builds.  How does that sound?